content/doc/developer/publishing/releasing-cd.adoc

```diff
@@ -71,6 +71,8 @@ IMPORTANT: These files may have been set up this way by the https://github.com/j
 
 === Configure Dependabot
 
+NOTE: If you have a `.github/renovate.json`, do not configure Dependabot. link:/blog/2023/09/20/renovate-bot-probe-blog[For more details]
```

Comment:
```suggestion
NOTE: If you have a `.github/renovate.json`, do not configure Dependabot. Refer to the following blog post link:/blog/2023/09/20/renovate-bot-probe-blog[for more details].
```
Adjusting the text so that it is a full sentence with a full stop.

Reply:
I can't see how this blog post is related it should not be linked here.

Something like:
```suggestion
NOTE: If you have link:https://docs.renovatebot.com/[Renovate] configured (normally in `renovate.json` or `.github/renovate.json`) do not configure Dependabot, as long as your renovate configuration isn't explicitly enabling a single manager renovate will automatically start updating your new dependencies that are being added here.
```

or a shorter one:

```suggestion
NOTE: If you have link:https://docs.renovatebot.com/[Renovate] configured (normally in `renovate.json` or `.github/renovate.json`) do not configure Dependabot.
```